mmdet/datasets/coco_panoptic.py

```diff
@@ -516,6 +620,14 @@ def evaluate(self,
                                                       outfile_prefix, logger,
                                                       classwise)
             eval_results.update(eval_pan_results)
+            metrics.remove('PQ')
+
+        if (('bbox' in metrics) or ('segm' in metrics)
+                or ('proposal' in metrics) or ('proposal_fast' in metrics)):
+            eval_ins_results = self.evaluate_ins_json(results, result_files,
```

Comment:
Can we use super().evaluate instead of renaming the evaluate function in CocoDataset?

Reply:
no, super().evaluate will call `CocoDataset.results2json` inside, but `results2json` is overwirted in cocopanoptic